vulture/core.py

```diff
@@ -184,13 +184,16 @@ def exclude(name):
             else:
                 self.scan(module_string, filename=module)
 
-            whitelist_names = ['stdlib.py']
-            for name in whitelist_names:
+            for name in self.defined_imports:
                 path = os.path.join('whitelists', name)
```

Comment:
She might, therefore we should add it. `--exclude sys` will ignore `sys.py` and `system.py`.